packages/core/docs/.vuepress/integrations.js

```diff
@@ -101,9 +101,9 @@ const INTEGRATIONS = {
     },
     {
       name: 'Sylius',
-      link: '',
+      link: 'https://github.com/vuestorefront-community/sylius',
```

Comment:
This should be a link to documentation, not a repository. The button with this link says "Read the docs".

Reply:
@filipsobol fixed